packages/core/README.md

```diff
@@ -89,7 +89,7 @@ core's tests. Running these specific tests can be done as follows:
 
```

Comment:
Not sure if intentional, but these fixes were left out:
```
In packages/core/test/oracle/VotingV2.js :
line 2300: "i.e" should be "i.e."
line 2481: "cant" should be "can't"
line 2482: "cant" should be "can't"
line 2630: "Dont" should be "Don't"
line 2708: "cant" should be "cant"
line 2815: "dont" should be "don't"
line 2857: "i.e" should be "i.e."
```

These aren't in the contract files, so it's not a huge deal if we don't fix these, but just wanted to mention.

Reply:
ye, I'm ok with unit tests not having this.